client/.env

```diff
@@ -0,0 +1,2 @@
+VITE_EMAIL=dunc@gmail.com
```

Comment:
dont push env files togithub- unless it is an example of how to use? otherwise remove this file